modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoverySystemViewTest.java

```diff
@@ -178,4 +186,21 @@ private static int getTableId(String schemaName, String tableName) {
 
         return catalogManager.catalog(catalogManager.latestCatalogVersion()).table(schemaName, tableName).id();
     }
+
+    private static long estimatedSize(String nodeName, String tableName, int partitionId) {
+        return CLUSTER.runningNodes()
+                .filter(ignite -> nodeName.equals(ignite.name()))
+                .map(ignite -> {
+                    IgniteImpl node = ((RestartProofIgnite) CLUSTER.node(0)).unwrap(IgniteImpl.class);
+
+                    TableImpl table = ((PublicApiThreadingTable) node.tables().table(tableName)).unwrap(TableImpl.class);
```

Comment:
You can just do

`TableImpl table = TestWrappers.unwrapTableImpl(ignite.tables().table(tableName))`